webui/react/src/services/api.ts

```diff
@@ -1,18 +1,11 @@
 import { CancelToken } from 'axios';
-import { sha512 }  from 'js-sha512';
 
-import { decode, ioTypeUser, ioUser } from 'ioTypes';
-import { Api, generateApi } from 'services/apiBuilder';
-import { jsonToExperiments } from 'services/decoder';
+import { generateApi } from 'services/apiBuilder';
+import * as AC from 'services/apiConfig';
```

Comment:
nit: `Config` instead of `AC`. Dropping the `Api` since it's already in the context of `Api` based on where it's referencing it.

Reply:
:+1:  as long as we don't have another config in here.